python/ray/serve/tests/test_proxy.py

```diff
@@ -329,58 +328,6 @@ async def test_consume_generator_stream(self):
         assert response.status_code == str(grpc_proxy.success_status_code)
         assert response.streaming_response is not None
 
-    @pytest.mark.asyncio
-    @patch("ray.serve._private.http_proxy.ray.get")
-    async def test_consume_generator_unary(self, mock_get):
```

Comment:
@GeneDer my code changes required me to change these tests to match the implementation details. I spent a few minutes trying this but it was too painful to update the mocks and fixtures. Let's update the tests to be less coupled to implementation details.

Reply:
Agreed, those tests took forever to setup. Will refactor and import in the near future.